### ISSUE TITLE
Message Sync: Deletes and New Adds (clarification)

### ISSUE THREAD

**Issue by grehce** [firebase/friendlychat-ios#13]
In Firebase Database, I added a name but made a misstep and added the text afterward. I later deleted the test_message; however, only the name shows up in the messenger thread on the simulator.

How are deletes and new field adds handled in the Friendly Chat app?

**Comment by morganchen12** [comment]
You can find the source for this [here](https://github.com/firebase/friendlychat-ios/blob/master/ios/objc/FriendlyChatObjC/FCViewController.m#L72), in the completed codelab code. The chat logic is extremely simplistic and only observes additions of new chat messages, capturing the snapshot at that time and adding it to an array of messages. This means if the message changes in the future or is removed, FriendlyChat will ignore it (since it's only listening to child added events, not child changed or child removed events).

In short, deletes and new field adds (modifications) aren't handled at all.

**Comment by grehce** [comment]
@morganchen12 thanks!
